README.md

```diff
@@ -108,30 +108,31 @@ hCaptcha.setup().verifyWithHCaptcha()
 1. The listeners (`onSuccess`, `onFailure`, `onOpen`) can be called multiple times in the following cases:
    1. the same client is used to invoke multiple verifications
    2. the config option `resetOnTimeout(true)` is used which will automatically trigger a new verification when the current token expired. This will result in a new success or error callback.
-
+   3. `onFailure` with `SESSION_TIMEOUT` will be called once the token is expired. To prevent this you can call `HCaptchaTokenResponse.markUsed` once the token is utilized. Also, you can change expiration timeout with `HCaptchaConfigBuilder.expirationTimeout(timeout)` (default 2 min.)
```

Comment:
"`onFailure` with `SESSION_TIMEOUT`" should be `TOKEN_TIMEOUT`.

Reply:
Fixed